### DISCUSSION TITLE
使用半自动工具标注时，图像中无目标该如何处理

### DISCUSSION THREAD

**Discussion by chenkunet** [PaddlePaddle/PaddleOCR#12510] [Q&A] [answered]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：linux
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：paddleocr-2.6.0
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

如题，使用半自动标注工具标注时，对于无目标文字的图片Label.txt中不会产生记录，导致图片无法作为负样本带入训练。
该如何解决这个问题？

**Comment by tink2123** [comment, marked as answer]
目前不支持标注负样本～
